### ISSUE TITLE
Should not use badge information for mobile browsers (since they don't have badges)

### ISSUE THREAD

**Issue by tehhowch** [DevJackSmith/mh-hunt-helper#102]
v19.3.9, connected in WebIDE, shows a lot of noise from the extension:

![image](https://user-images.githubusercontent.com/20871346/54298437-bec49800-4586-11e9-94e1-d0d36a4b1fa0.png)

We should avoid attempting to update the badge text for the mobile browser.

**Comment by DevJackSmith** [comment]
yeah, agree, we should review the whole extension for errors on Android and put check(s) where needed.
